account_cash_invoice/models/account_move.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Creu Blanca
+# Copyright (C) 2024 Tecnativa <https://tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _post(self, soft=True):
```

Comment:
Seeing this, please avoid this override that is anti-performant. Do the reconciliation directly from the wizard for the specific lines.